src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -0,0 +1,168 @@
+/* ==========================================================================
+   jquery.cf_notifier
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var handlebars = require( 'handlebars' );
+
+var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" style="display: none;">' +
+  '<span class="cf-notification_icon cf-notification_icon__{{ state }} cf-icon cf-icon-{{ icon }}-round"></span>' +
+  '<p class="cf-notification_text">{{ message }}</p>' +
+  '</div>';
+
+var _notifier = {
```

Comment:
Notifier should be moved to its own module so there aren't two init functions being declared in the same file.